plugins/metrics/src/main/java/org/apache/cloudstack/metrics/MetricsServiceImpl.java

```diff
@@ -712,7 +712,7 @@ public List<HostMetricsResponse> listHostMetrics(List<HostResponse> hostResponse
             metricsResponse.setCpuTotal(hostResponse.getCpuNumber(), hostResponse.getCpuSpeed());
             metricsResponse.setCpuUsed(hostResponse.getCpuUsed(), hostResponse.getCpuNumber(), hostResponse.getCpuSpeed());
             metricsResponse.setCpuAllocated(hostResponse.getCpuAllocated(), hostResponse.getCpuNumber(), hostResponse.getCpuSpeed());
-            metricsResponse.setLoadAverage(hostResponse.getAverageLoad());
+            metricsResponse.setCpuAverageLoad(hostResponse.getAverageLoad());
```

Comment:
HostResponse class has `setCpuAverageLoad` to set the value `cpuloadaverage` in response.